manifests/cf-manifest/operations.d/730-elasticache-broker.yml

```diff
@@ -93,6 +99,16 @@
                   cluster-enabled: 'no'
                   maxmemory-policy: volatile-lru
                   reserved-memory: '0'
+              b6949ea7-5c98-4c69-b981-4b5318ea8b7c:
+                instance_type: cache.m5.large
+                replicas_per_node_group: 2
+                shard_count: 1
+                snapshot_retention_limit: 7
+                automatic_failover_enabled: true
+                parameters:
+                  cluster-enabled: 'yes'
```

Comment:
I don't think this is what we want. I think what we want for this plan is non-clustered Redis with a single replia to allow automatic failover.

Cluster mode looks to be about having multiple node groups, each with a master and replicas for use when you're sharding things (https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/Replication.Redis-RedisCluster.html). IIRC the only reason we enabled cluster mode for the initial plan was because AWS don't support backups for non-clustered t2.micro instances. Clustered mode doesn't play nicely with things like Sidekiq, hence we had to create the non-backed up unclustered plan.

Reply:
I note that since we initially created these plans(2b9da87), the limitations seems to have changed, and it's only `cache.t1.micro` instances that the restriction applies to now (https://docs.aws.amazon.com/AmazonElastiCache/latest/red-ug/backups.html#backups-constraints). I'm going to raise a story to look at enabling backups for both plans (and perhaps deprecating the clustered plan).